test/unit/metrics_plots/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Comment:
not sure if this file is required.

Reply:
Turns out it is because we are importing from conftest which otherwise can't be found.

Reply:
This is now removable after merging all the content as indicated in https://github.com/fairlearn/fairlearn/pull/561/files/84ae2abcde883726545c2d11d3cd18a5b9b6633f#r551555912 which I now realize is probably what you wanted all along.